fenix/app/src/test/java/org/mozilla/fenix/components/appstate/ShoppingActionTest.kt

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.components.appstate
+
+import mozilla.components.support.test.ext.joinBlocking
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mozilla.fenix.components.AppStore
+import org.mozilla.fenix.components.appstate.shopping.ShoppingState
+
+class ShoppingActionTest {
+
+    @Test
+    fun `WHEN shopping sheet is expanded THEN state should reflect that`() {
```

Comment:
the test names are flipped. this one should be collapsed the next one should be expanded.

Reply:
Very true!